src/main/java/mate/academy/service/UserService.java

```diff
@@ -0,0 +1,11 @@
+package mate.academy.service;
+
+import java.util.Optional;
+import mate.academy.model.User;
+
+public interface UserService {
+
```

Comment:
```suggestion
```
Don’t  leave an empty line after the class name or make it consistent for each class